lib/osf-components/addon/components/node-metadata-form/template.hbs

```diff
@@ -0,0 +1,87 @@
+<h2>
+    {{@manager.node.title}}
+    {{#if @manager.userCanEdit}}
+        <Button
+            data-test-edit-metadata-button
+            aria-label={{t 'general.edit'}}
+            data-analytics-name='Edit metadata'
+            {{on 'click' @manager.edit}}
+        >
+            {{t 'osf-components.node-metadata-form.edit-button'}}
+        </Button>
+    {{/if}}
+</h2>
+<div>{{@manager.node.description}}</div>
+{{#if @manager.inEditMode}}
+    <Button
+        data-test-cancel-editing-metadata-button
+        aria-label={{t 'general.cancel'}}
+        data-analytics-name='Cancel editing metadata'
+        {{on 'click' @manager.cancel}}
+    >
+        {{t 'general.cancel'}}
+    </Button>
+    <Button
+        data-test-save-metadata-button
+        aria-label={{t 'general.save'}}
+        data-analytics-name='Save metadata'
+        {{on 'click' @manager.save}}
+    >
+        {{t 'general.save'}}
+    </Button>
+{{else}}
+    <div>
+        <div>{{t 'osf-components.node-metadata-form.resource-information'}}</div>
+        <div>
+            <span>{{t 'osf-components.node-metadata-form.resource-type'}}</span>
+            <span>{{@manager.metadataRecord.resourceTypeGeneral}}</span>
+        </div>
+        <div>
+            <span>{{t 'osf-components.node-metadata-form.resource-language'}}</span>
+            <span>{{@manager.metadataRecord.language}}</span>
+        </div>
```

Comment:
I think this can be written more semantically as:
```suggestion
        <dl>
            <dt>{{t 'osf-components.node-metadata-form.resource-type'}}</dt>
            <dd>{{@manager.metadataRecord.resourceTypeGeneral}}</dd>
            <dt>{{t 'osf-components.node-metadata-form.resource-language'}}</dt>
            <dd>{{@manager.metadataRecord.language}}</dd>
        </dl>
```